assets/sass/components/settings/_googlesitekit-settings-module.scss

```diff
@@ -179,6 +179,10 @@
 			}
 		}
 
+		.googlesitekit-settings-module__fields-group-title {
+			font-size: $fs-body-md;
```

Comment:
Sorry for not noticing this earlier. While implementing a design from Figma, the key point is to use the "token" mentioned in the element typography instead of using the font-size specified. For example:

![image](https://user-images.githubusercontent.com/20284937/205862697-00aa0491-584c-409d-ab58-6c0b7eee290c.png)

This element uses the `title/small` token, for which, we need to use the `$fs-title-sm` variable to specify the font size.

Also, this title has a font-weight of `500`. Could you update it to have a font-weight of `$fw-medium`?

```suggestion
		.googlesitekit-settings-module__fields-group-title {
			font-size: $fs-title-sm;
			font-weight: $fw-medium;
```